problems/arrayMethods.js

```diff
@@ -91,15 +143,31 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+  let object = {};
+  items.map((el) => {
+    if (object[el]) {
+      return (object[el] += 1);
+    } else {
+      return (object[el] = 1);
+    }
+  });
+  return object;
+};
 
 /**
  * Takes in an array of numbers and returns a new array with each element doubled.
  * @param {number[]} nums
  * @returns {number[]} Each element has been doubled
  */
 
-const doubled = () => {};
+const doubled = (nums) => {
+  let output = [];
+  nums.forEach((el) => {
```

Comment:
The right array method for this is a map since we are only changing the elements in num.